articles/evaluation_queues.md

```diff
@@ -0,0 +1,124 @@
+	submissionReceiptMessage="Thanks for submitting to My Example Challenge!")
+syn.store(evaluation)
+		{% endhighlight %}
+	{% endtab %}
+
+	{% tab R %}
+		{% highlight r %}
+evaluation <- Evaluation(name="My Example Challenge",
+	description="Short description of challenge queue",
+	status="OPEN",
+	quota=synapseClient:::SubmissionQuota(submissionLimit=3, #The maximum number of submissions per team/participant per round.
+		firstRoundStart = '2017-11-02T07:00:00.000Z', #The date/time at which the first round begins in UTC.
+		roundDurationMillis = 1645199000, #The duration of each round.
+		numberOfRounds=1), #	The number of rounds, or null if there is no end. (Based on the duration of each round)
+	contentSource="syn12345", #Your synapse project id
+	submissionInstructionsMessage="Instructions on submission format...",
+	submissionReceiptMessage="Thanks for submitting to My Example Challenge!")
+synStore(evaluation)
+		{%endhighlight %}
+	{% endtab %}
+
+	{% tab Web %}
+You can create evaluation queues on the web by navigating to your challenge site by adding `/admin` to the url (E.g. www.synapse.org/#!Synapse:syn12345/admin).  Click **Tools** on the right corner and **Add Evaluation Queue** and follow the instructions on the screen.
+
+<img src="/assets/images/create_evaluation_queues.png">
+
+	{% endtab %}
+{% endtabs %}
+
+## Configure an Evaluation Queue 
+
+Unforuntately there is not a way to configure the round or quota settings of an evaluation queue from the web. The evaluation id can be found under the **Challenge** tab.  In the case below, the evaluation queue id is 9610091.  
+
+<img style="width: 80%;" src="/assets/images/evaluation_queue_id.png">
+
+Using this value, we can configure the `quota` parameters of this evaluation queue with the R or python client.  
+
+{% tabs %}
+	{% tab Python %}
+		{% highlight python %}
+evalId = 9610091
+evaluation = syn.getEvaluation(evalId)
+evaluation.quota = {'submissionLimit':3} #The maximum number of submissions per team/participant per round.
+syn.store(evaluation)
+		{% endhighlight %}
+	{% endtab %}
+
+	{% tab R %}
+		{% highlight r %}
+evalId = 9610091
+evaluation <- synGetEvaluation(evalId)
+evaluation$quota = synapseClient:::SubmissionQuota(firstRoundStart = '2017-11-02T07:00:00.000Z', #The date/time at which the first round begins in UTC.
+	roundDurationMillis = 1645199000, #The duration of each round.
+	numberOfRounds=1) #	The number of rounds, or null if there is no end. (Based on the duration of each round) 
+synStore(evaluation)
+		{%endhighlight %}
+	{% endtab %}
+
+{% endtabs %}
+
+
+## Share an Evaluation Queue
+
+Each Evaluation has its own sharing settings.  The sharing levels are: "Administrator", "Can score", "Can Submit", and "Can view."  
+- "Administrator" sharing should be tightly restricted, as it includes authority to delete the entire Evaluation queue with all its contents. These users also have the ability to download all the submissions.
```

Comment:
They can download even in the presence of sharing settings or access restrictions that may not allow?

Reply:
Submission objects are weird.  All file entities submitted are automatically "duplicated" in synapse.  So yes, all the submissions are always available.

Reply:
I understand why - is that documented explicitly anywhere? We should be clear that sharing settings go out the window when submitting to a Evaluation queue.

Reply:
I added a statement about this.